superset-frontend/src/explore/ExplorePage.tsx

```diff
@@ -117,10 +117,12 @@ export default function ExplorePage() {
     if (!isExploreInitialized.current || isSaveAction) {
       fetchExploreData(exploreUrlParams)
         .then(({ result }) => {
-          const formData = getFormDataWithDashboardContext(
-            result.form_data,
-            dashboardContextFormData,
-          );
+          const formData = dashboardContextFormData
+            ? getFormDataWithDashboardContext(
+                result.form_data,
+                dashboardContextFormData,
+              )
+            : result.form_data;
```

Comment:
Could this be merged into `getFormDataWithDashboardContext`? Basically make the second parameter nullable?

Reply:
I'd rather to keep this way because using `getFormDataWithDashboardContext` without dashboardContent seems odd.